serving/docs/lmi/user_guides/output_formatter_schema.md

```diff
@@ -0,0 +1,97 @@
+# Custom output formatter schema
+
+This document provides the schema of the output formatter, with which you can write your own custom output formatter. 
+
+
+## Signature of your own output_formatter
+
+To write your own custom output formatter, follow the signature below:
+```
+from djl_python.output_formatter import RequestOutput
+
+def custom_output_formatter(request_output: RequestOutput) -> str:
+    #your implementation here
+```
+
+## RequestOutput schema
+The RequestOutput class is designed to encapsulate the output of a request in a structured format. Here is an in-depth look at its structure and the related classes:
+```mermaid
+classDiagram
+    RequestOutput <|-- TextGenerationOutput
+    RequestOutput *-- RequestInput
+    TextGenerationOutput "1" --> "1..*" Sequence
+    Sequence "1" --> "1..*" Token
+    RequestInput <|-- TextInput
+    class RequestOutput{
+        +int request_id
+        +bool finished
+        +RequestInput input
+    }
+    class TextGenerationOutput{
+        +map sequences
+        +int best_sequence_index
+        +list[Token] prompt_tokens_details
+    }
+    class Sequence{
+        +list[Token] tokens
+        +float cumulative_log_prob
+        +string finish_reason
+        +has_next_token()
+        +get_next_token()
+    }
+    
+    class RequestInput{
+        +int request_id
+        +dict parameters
```

Comment:
we used map previously, and dict here. Let's keep it consistent (or maybe i'm not understanding the difference between the two).

If they are the same, should we say what the mapping is? (from what to what)